src/shared/utils/prismLanguageMapper.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Maps file extension names to prism supported languages.
+ * The full list can be found in the prism-react-renderer package, just cmd +
+ * click on the import string.
```

Comment:
Not everyone has the same editor or key bindings and it is better to add a source code link to the line in gh